config/quickwit.yaml

```diff
@@ -80,9 +80,12 @@ version: 0.6
 #     access_key: ${QW_AZURE_STORAGE_ACCESS_KEY}
 #
 #   s3:
+#     access_key_id: ${AWS_ACCESS_KEY_ID}
+#     secret_access_key: ${AWS_SECRET_ACCESS_KEY}
 #     endpoint: ${QW_S3_ENDPOINT}
-#     force_path_style_access: ${QW_S3_FORCE_PATH_STYLE_ACCESS:-true}
-#     disable_multi_object_delete_requests: true
+#     region: ${AWS_REGION}
+#     force_path_style_access: ${QW_S3_FORCE_PATH_STYLE_ACCESS:-false}
+#     disable_multi_object_delete_requests: false
```

Comment:
I'd rather leave the value to `true`. If you are uncommenting this line, you most likely want to change the default behavior.

Reply:
From a new user's perspective if I see an option in the example - I assume that's the recommended and default value - however in this case I then read the documentation and it turns out that the docs say the default is something different.